### PR TITLE
Sprin 19 transfer to event banner

### PR DESCRIPTION
**Pull request by mykolazubchyk**
With a bug. 
[Bug review](https://vitaliyosyp9282.atlassian.net/jira/software/projects/SPRIN/boards/3?assignee=712020%3A9a14d7b2-a03a-4f79-a4cf-24f8080815db&selectedIssue=SPRIN-124) 
[Test case](https://vitaliyosyp9282.atlassian.net/jira/software/projects/SPRIN/boards/3?assignee=712020%3A9a14d7b2-a03a-4f79-a4cf-24f8080815db&selectedIssue=SPRIN-19)

### CODE REVIEW

File: src/main/java/pages/OldMeetsKnuPage.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import fragments.HeaderFragment;
+import org.openqa.selenium.WebDriver;
+
+import java.util.List;
+
+import static common.CommonActions.getListOfString;
+
+public class OldMeetsKnuPage extends HeaderFragment {
+
```
Comment (KhraschukZahar) [inline src/main/java/pages/OldMeetsKnuPage.java:11]: you should extend BasePage

File: src/main/java/pages/Homepage.java [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+        waitClickabilityOfElement(EVENT_BANNER_ONE).click();
+    }
+
+    public void eventBannerTwoClick() {
+        waitClickabilityOfElement(EVENT_BANNER_TWO).click();
+    }
+
+    public void eventBannerThreeClick() {
+        waitClickabilityOfElement(EVENT_BANNER_THREE).click();
+    }
```
Comment (KhraschukZahar) [inline src/main/java/pages/Homepage.java:31]: naming

File: src/main/java/pages/UltraRangePage.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package pages;
+
+import fragments.HeaderFragment;
+import org.openqa.selenium.WebDriver;
+
+import java.util.List;
+
+import static common.CommonActions.*;
+
+public class UltraRangePage extends HeaderFragment {
```
Comment (KhraschukZahar) [inline src/main/java/pages/UltraRangePage.java:10]: you should extend BasePage

File: src/main/java/pages/TheAvePage.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package pages;
+
+import fragments.HeaderFragment;
+import org.openqa.selenium.WebDriver;
+
+import java.util.List;
+
+import static common.CommonActions.getListOfString;
+
+public class TheAvePage extends HeaderFragment {
```
Comment (KhraschukZahar) [inline src/main/java/pages/TheAvePage.java:10]: you should extend BasePage

File: src/test/java/tests/EventBannersTest.java [hunk 5]
```diff
@@ -59,7 +59,7 @@ public void eventBannersContent() {
         }
 
         goToPreviousPage(driver);
-        homepage.eventBannerThreeClick();
+        homepage.thirdEventBannerGet();
```
Comment (KhraschukZahar) [inline src/test/java/tests/EventBannersTest.java:62]: the word "get" should come first, call methods what the others contributors call it

**Review by KhraschukZahar — APPROVED** [review]

**Review by Goldbe54 — COMMENTED** [review]
to be continued

File: src/test/java/tests/EventBannersTest.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+import static common.CommonActions.goToPreviousPage;
+import static common.CommonActions.scrollToElement;
+
+public class EventBannersTest extends BaseTest {
+
+    public static Homepage homepage;
+    public static UltraRangePage ultraRangePage;
+    public static BmxCollectionPage bmxCollectionPage;
+    public static TheAvePage theAvePage;
```
Comment (Goldbe54) [inline src/test/java/tests/EventBannersTest.java:21]: make it `private` and `final`

File: src/test/java/tests/EventBannersTest.java [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    public void initTest() {
+        homepage = new Homepage(driver);
+        ultraRangePage = new UltraRangePage(driver);
+        theAvePage = new TheAvePage(driver);
+        bmxCollectionPage = new BmxCollectionPage(driver);
+    }
+
+    @Issue("SPRIN-124")
+    @Test(description = "Transfer to event banners and checking their content")
+    public void eventBannersContent() {
```
Comment (Goldbe54) [inline src/test/java/tests/EventBannersTest.java:33]: This is test method, so rename it to test method

File: src/main/java/common/CommonActions.java [hunk 8]
```diff
@@ -111,4 +110,16 @@ public static void scrollToElement(WebDriver driver, WebElement element) {
...
+        driver.navigate().back();
+    }
+
+    public static List<String> convertListToLowerCase(List<String> products) {
+        List<String> lowerCaseProducts = new ArrayList<>();
+        for (String product : products) {
+            lowerCaseProducts.add(product.toLowerCase());
+        }
+        return lowerCaseProducts;
+    }
```
Comment (Goldbe54) [inline src/main/java/common/CommonActions.java:124]: Try use stream in this method

**Review by Goldbe54 — APPROVED** [review]

**Pull request merged**